scene/animation/tween.cpp

```diff
@@ -844,3 +865,49 @@ MethodTweener::MethodTweener(const Callable &p_callback, const Variant &p_from,
 MethodTweener::MethodTweener() {
 	ERR_FAIL_MSG("MethodTweener can't be created directly. Use the tween_method() method in Tween.");
 }
+
+void SubtweenTweener::start() {
+	finished = false;
+
+	// Reset the subtween.
+	subtween->stop();
+	subtween->play();
+}
+
+bool SubtweenTweener::step(double &r_delta) {
+	if (finished) {
+		return false;
+	}
+
+	elapsed_time += r_delta;
+
+	if (elapsed_time < delay) {
+		r_delta = 0;
+		return true;
+	}
+
+	if (!subtween->step(r_delta)) {
+		_finish();
```

Comment:
```suggestion
		r_delta = elapsed_time - delay - subtween->get_total_time();
		_finish();
```
Last step might not use the whole delta, so the remainder should be calculated.